user/managing-os/xfce-templates.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: doc
+title: Xfce TemplateVMs
+permalink: /doc/templates/xfce/
+redirect_from:
+- /doc/templates/xfce/
+- /doc/xfce/
+- /en/doc/templates/xfce/
+- /doc/Templates/Xfce/
+- /wiki/Templates/Xfce/
+---
+
```

Comment:
@andrewdavidwong I've read the modifications do you see anything wrong with the above modifications of Fedora Xfce into a more general cathegory like the "Minimal" one?

Reply:
I was thinking the same thing. Sounds like a good move to me. :slightly_smiling_face: